### ISSUE TITLE
implement problems?

### ISSUE THREAD

**Issue by Dian-Yi** [chloeguoqing/Towards-Open-Set-Identity-Preserving-Face-Synthesis#5]
i use your code implement this paper, first i change u code mang parts to make it can train. but when i train the code, there are mang problems:
1. loss GD , loss GR, loss GC,loss KL dont descend， it will ascend and Shock
2. the result image is bad

so, i doubt this paper cant balance the different loss when train? Only loss I and loss D descend when train because they have label. Unsupervised model examples network G and A is hard to train thought they have Seemingly reasonable loss function.

**Comment by vnyk** [comment]
add a '-' sign in front of the equations

**Comment by iperov** [comment]
i am trying to implement

![python_2019-11-19_11-00-46](https://user-images.githubusercontent.com/8076202/69123852-d6600300-0abb-11ea-9ca3-7aeaacfadad6.jpg)

**Comment by Dian-Yi** [comment]
> i am trying to implement
> 
> ![python_2019-11-19_11-00-46](https://user-images.githubusercontent.com/8076202/69123852-d6600300-0abb-11ea-9ca3-7aeaacfadad6.jpg)

How is the final result?

**Comment by iperov** [comment]
@Dian-Yi 
no good result.
The paper is fake ( as 70% of papers in arxiv ).

**Comment by ainrichman** [comment]
> @Dian-Yi
> no good result.
> The paper is fake ( as 70% of papers in arxiv ).

I agree with you. It is impossible to reconstruct the identity details with only 1 dimension latent vector. It is also impossible that the weights of the decoder could keep the appearance details of million faces. The images provided by the paper must be manually edited.

**Comment by xuefeng-li-sydney** [comment]
> > @Dian-Yi
> > no good result.
> > The paper is fake ( as 70% of papers in arxiv ).
> 
> I agree with you. It is impossible to reconstruct the identity details with only 1 dimension latent vector. It is also impossible that the weights of the decoder could keep the appearance details of million faces. The images provided by the paper must be manually edited.

Is there any way we could complain about this paper? It wasted me about two months of time to compare my method with theirs.

**Comment by Chenzhangwu28746** [comment]
@iperov  Can you provide the code that you implement？

**Comment by zideliu** [comment]
> i am trying to implement
> 
> ![python_2019-11-19_11-00-46](https://user-images.githubusercontent.com/8076202/69123852-d6600300-0abb-11ea-9ca3-7aeaacfadad6.jpg)


I would like to ask if you have reproduced it. Recently, I was reading this paper and found that it is a bit difficult to implement. I wonder if I can read your code. My email is 1281663983@qq.com

**Comment by zideliu** [comment]
> i am trying to implement
> 
> ![python_2019-11-19_11-00-46](https://user-images.githubusercontent.com/8076202/69123852-d6600300-0abb-11ea-9ca3-7aeaacfadad6.jpg)

Have you successfully implemented this paper? If so, could you please share your repositories? I reeally need it

**Comment by zideliu** [comment]
> 

Recently, I also implemented this paper. I also encountered the same problem as you

**Comment by iperov** [comment]
this is shit paper as many others.
